metashare/api/sf_run_flow.py

```diff
@@ -169,6 +169,7 @@ def get_org_result(
                 else None
             ),
             "OrgName": scratch_org_definition.get("orgName", "MetaShare Task Org"),
+            "SignupInstance": "cs68",
```

Comment:
Reminder: don't merge this

Reply:
Thanks for the reminder. Any idea when the upstream problem will be fixed?

Reply:
Someone told us today, but I don't have any confidence in that. I'd recommend turning this into something that can be set optionally via an env var.